front/src/styles/scss/common/map/_mapSearch.scss

```diff
@@ -0,0 +1,93 @@
+
+.map-modal {
+  .search-results-label {
+    cursor: pointer;
+  }
+  .search-results {
+    height: 30vh;
+    overflow: auto;
+    padding-right: 0.5rem;
+    .operational-point {
+      all: unset;
+      display: flex;
+      align-items: center;
+      padding: 0 0.25rem;
+      border-radius: var(--border-radius);
+      width: 100%;
+      cursor: pointer;
+      line-height: 1.5rem;
+      .trigram {
+        font-size: 0.7rem;
+        width: 1.75rem;
+      }
+      .name {
+        padding-left: 0.5rem;
+        .ch {
+          margin-left: 0.25rem;
+          font-size: 0.9rem;
+        }
+      }
+      .uic {
+        margin-left: auto;
+        font-size: 0.7rem;
+        color: var(--primary);
+      }
+      &.main {
+        .name {
+          font-size: 1.1rem;
+          padding-left: 0;
+          font-weight: 500;
+        }
+      }
```

Comment:
Sometimes the spacing between the trigram and the name is very short

![Capture d’écran 2024-02-22 à 09 17 58](https://github.com/osrd-project/osrd/assets/113104137/a03cac00-3276-4f2b-9d76-814af02ccbe4)

Reply:
We could use, according to your recommendation, monospace font.

Reply:
Done!